android-components/components/concept/engine/src/main/java/mozilla/components/concept/engine/translate/TranslationOperation.kt

```diff
@@ -24,4 +24,9 @@ enum class TranslationOperation {
      * and localized name.
      */
     FETCH_LANGUAGES,
+
+    /**
+     * The page related settings the translation engine should fetch.
+     */
+    FETCH_PAGE_SETTINGS,
```

Comment:
Flagging for discussion - I'm thinking the `Fetch` `Operations` should be here and not a new object, so we can reuse the error setting reducer. 

I could see them being their own type on the basis that there could be a new `Fetch` type. Open to either.